go.mod

```diff
@@ -1,18 +1,14 @@
 module github.com/opencontainers/image-spec
 
-go 1.11
+go 1.16
```

Comment:
I think I'm okay with this because we just cut a release that older versions of go can continue to use, but I can imagine this being controversial.

Reply:
Yes, that's a good call out but is a requirement to use `go:embed`. If we don't want to move the req up to 1.16, we should leave esc in place.

Reply:
I did a spot check of dependencies and it looks like most are 1.16, some at 1.17, but there are some projects still at 1.14 and 1.15. Given that 1.15 is no longer officially supported by the Go, I'm okay with this myself.